backend/tilavarauspalvelu/models/user/model.py

```diff
@@ -120,7 +120,15 @@ def active_general_permissions(self) -> list[UserPermissionChoice]:
 
     @property
     def active_unit_roles(self) -> dict[int, list[UserRoleChoice]]:
-        """Get unit roles by unit id for the user."""
+        """
+        Get unit roles by unit id for the user.
+
+        Output format:
+        {
+            1: [UserRoleChoice.RESERVER, UserRoleChoice.VIEWER],
+            2: [UserRoleChoice.NOTIFICATION_MANAGER],
```

Comment:
Better use `HANDLER` as the third example (here and in the tests), since `NOTIFICATION_MANAGER` will be restricted to a general role in [TILA-4277](https://helsinkisolutionoffice.atlassian.net/browse/TILA-4277)